bot/exts/utilities/conversationstarters.py

```diff
@@ -63,7 +65,24 @@ def _build_topic_embed(channel_id: int) -> discord.Embed:
             embed.title = f"**{next(TOPICS['default'])}**"
         else:
             embed.title = f"**{next(channel_topics)}**"
-        return embed
+
+        if previous_topic is None:
+            # This is the first topic being sent
+            return embed, False
+
+        total_topics = previous_topic.count("\n") + 1
+        # Add 1 before first topic
+        if total_topics == 1:
+            previous_topic = f"1. {previous_topic}"
+
+        embed.title = previous_topic + f"\n{total_topics + 1}. {embed.title}"
+
+        # When the embed will be larger than the limit, use the previous embed instead
```

Comment:
The comment seems wrong and unnecessary. It is wrong because the comment talks about the embed as a whole, while we are operating on embed's title. It is unnecessary because it can be inferred from the code that we reassign the `embed.title` if it exceeds the title character limit